doc/programming/language/packing.rst

```diff
@@ -0,0 +1,57 @@
+
+.. _packing:
+
+==========================
+Packing / Unpacking Values
+==========================
+
+A few of Spicy's atomic types support low-level conversion from, or
+into, a binary representation through two operators:
```

Comment:
Regarding the byte order argument to `pack` and `unpack`, for parsing there is an implicit byte order. Do you think it would make sense to also have a default byte order for `pack` and `unpack`?

Reply:
I prefer leaving this explicit. It's pretty low-level functionality, so I think we can require people to specify what they want here for clarity.